lib/transfer_util.cu

```diff
@@ -704,23 +702,24 @@ namespace quda {
 
   template<typename Float>
   void BlockOrthogonalize(ColorSpinorField &V, int Nvec, 
```

Comment:
Added, verified things still compile and work! Thanks for noticing it could be done there too.